ros:
  bag: "/data/euroc/rosbags/imu_april.bag"
  cam0_topic: "/cam0/image_raw"
  cam1_topic: "/cam1/image_raw"
  imu0_topic: "/imu0"

settings:
  data_path: "/tmp/imu_april"
  results_path: "/tmp/imu_april/calib_results.yaml"

calib_target:
  target_type: 'aprilgrid'  # Target type
  tag_rows: 6               # Number of rows
  tag_cols: 6               # Number of cols
  tag_size: 0.088           # Size of apriltag, edge to edge [m]
  tag_spacing: 0.3          # Ratio of space between tags to tagSize
                            # Example: tagSize=2m, spacing=0.5m --> tagSpacing=0.25[-]

imu0:
  rate: 200.0
  sigma_a_c: 0.002
  sigma_g_c: 1.6968e-04
  sigma_aw_c: 0.003
  sigma_gw_c: 1.9393e-05
  g: 9.81007

cam0:
  resolution: [752, 480]
  proj_model: "pinhole"
  dist_model: "radtan4"
  proj_params: [458.654, 457.296, 367.215, 248.375]
  dist_params: [-0.28340811, 0.07395907, 0.00019359, 1.76187114e-05]

cam1:
  resolution: [752, 480]
  proj_model: "pinhole"
  dist_model: "radtan4"
  proj_params: [457.587, 456.134, 379.999, 255.238]
  dist_params: [-0.28368365, 0.07451284, -0.00010473, -3.555e-05]

T_cam0_cam1:
  rows: 4
  cols: 4
  data: [0.999997256478, -0.002317135723, -0.000343393121,  0.110074137800,
         0.002312067192,  0.999898048507, -0.014090668453, -0.000156612054,
         0.000376008102,  0.014089835847,  0.999900662638,  0.000889382785,
         0.000000000000,  0.000000000000,  0.000000000000,  1.000000000000]
